### PR TITLE
Added validation for boolean filters

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: business_register/filters.py [hunk 1]
```diff
@@ -75,7 +76,8 @@ class PepFilterSet(filters.FilterSet):
     fullname = filters.CharFilter(lookup_expr='icontains')
     fullname_transcriptions_eng = filters.CharFilter(lookup_expr='icontains')
     updated_at = filters.DateFromToRangeFilter()
-
+    is_pep = ValidationBooleanFilter()
+    is_dead = ValidationBooleanFilter()
```
Comment (roman-tiukh) [inline business_register/filters.py:80]: remove this fields from `Meta.fields`

File: data_ocean/filters.py [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
 
 
+class ValidationBooleanField(forms.NullBooleanField):
+
+    def validate(self, value):
+        if str(value) not in ('true', 'false', '1', '0', 'True', 'False'):
+            raise ValidationError(
+                {self.label:
+                     _('Invalid request parameter: could be True, False, true, false, 1, 0')}
+            )
```
Comment (roman-tiukh) [inline data_ocean/filters.py:16]: add current  invalid value for message
```suggestion
            raise ValidationError({
                self.label: _(f'Invalid request parameter: "{value}", could be True, False, true, false, 1, 0'),
            })
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/filters.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/filters.py]: done

File: data_ocean/filters.py (hunk 2 above)
Comment (IegorSoboliev) [reply data_ocean/filters.py]: value in form already is cleaned. So moved the whole process to a custom widget again

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**